openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -66,21 +65,43 @@ def setUp(self):
         self.client.login(username=self.user.username, password=TEST_PASSWORD)
 
     def tearDown(self):
-        remove_course_updates(self.course)
+        remove_course_updates(self.user, self.course)
         super(TestCourseHomePage, self).tearDown()
 
     @override_waffle_flag(UNIFIED_COURSE_TAB_FLAG, active=True)
     def test_welcome_message_when_unified(self):
+        # Create a welcome message
+        create_course_update(self.course, self.user, TEST_WELCOME_MESSAGE)
+
         url = course_home_url(self.course)
         response = self.client.get(url)
         self.assertContains(response, TEST_WELCOME_MESSAGE, status_code=200)
 
     @override_waffle_flag(UNIFIED_COURSE_TAB_FLAG, active=False)
     def test_welcome_message_when_not_unified(self):
+        # Create a welcome message
+        create_course_update(self.course, self.user, TEST_WELCOME_MESSAGE)
+
         url = course_home_url(self.course)
         response = self.client.get(url)
         self.assertNotContains(response, TEST_WELCOME_MESSAGE, status_code=200)
 
+    @override_waffle_flag(UNIFIED_COURSE_TAB_FLAG, active=True)
+    def test_updates_tool_visibility(self):
+        """
+        Verify that the updates course tool is visible only when the course
+        has one or more updates.
+        """
+        remove_course_updates(self.user, self.course)
```

Comment:
This shouldn't be necessary now that you've removed the welcome message in the setUp method.